### PR TITLE
slzb-06m support added

### PR DESCRIPTION
**Pull request by gam-st**

### CODE REVIEW

File: assets/manifests/slzb-06m.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"firmwares": [
+		{
+			"name": "Zigbee (EZSP)",
+			"url": "https://raw.githubusercontent.com/darkxst/silabs-firmware-builder/main/firmware_builds/slzb-06m/ncp-uart-hw-v7.4.1.0-slzb-06m-115200.gbl",
+			"type": "ncp-uart-hw",
+			"version": "7.4.1.0"
+		},
+		{
+			"name": "Multi-PAN (RCP)",
+			"url": "https://raw.githubusercontent.com/darkxst/silabs-firmware-builder/main/firmware_builds/slzb-06m/rcp-uart-802154-v4.4.1-slzb-06m-460800.gbl",
```
Comment (tl-sl) [inline assets/manifests/slzb-06m.json:24]: Multipan firmware must use v7.3.1 for the time being. All firmware since then are broken.

**Review by tl-sl — CHANGES_REQUESTED** [review]
Just the one comment to fix up, the rest looks good, thanks!

File: assets/manifests/slzb-06m.json (hunk 1 above)
Comment (gam-st) [reply assets/manifests/slzb-06m.json]: corrected here https://github.com/darkxst/silabs-firmware-builder/pull/64

Comment (gam-st) [reply assets/manifests/slzb-06m.json]: corrected

**Pull request merged**